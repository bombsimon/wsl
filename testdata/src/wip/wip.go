package testpkg

import "fmt"

func fn1() {
	one := 1 // want "missing whitespace decreases readability"

	if 1 == 1 {
		fmt.Println(1)
	} else if 1 == 2 {
		fmt.Println(2)
	} else {
		fmt.Println(2)
	}

	_ = one
}
