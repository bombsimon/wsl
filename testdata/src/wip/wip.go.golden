
package testpkg

func fn1() {
	two := 2

	one := 1

	if two == 2 { // want "missing whitespace decreases readability"
		panic(err)
	}

	_ = one
}
